airflow/gcp/hooks/gcs.py

```diff
@@ -289,7 +309,6 @@ def delete(self, bucket_name, object_name):
     def list(self, bucket_name, versions=None, max_results=None, prefix=None, delimiter=None):
         """
         List all objects from the bucket with the give string prefix in name
-
```

Comment:
This change prevents the documentation from being built properly. Can you delete it?